libs/map/src/lib/components/action/vl-map-click-action.ts

```diff
@@ -0,0 +1,78 @@
+import {webComponent} from '@domg-wc/common-utilities';
+import {VlMap} from '@domg-wc/map';
+import {LitElement} from "lit";
+import {MapBrowserEvent} from "ol";
+import Overlay from "ol/Overlay";
+import {property} from 'lit/decorators.js';
+import {Coordinate} from "ol/coordinate";
+import {Extent} from "ol/extent";
+import {VlMapClickActionPindrop} from "./vl-map-click-action-pindrop";
+import {Pixel} from "ol/pixel";
+
+/**
+ * VlMapClickAction
+ * @classdesc The map click action component. Adds a marker on the map.
+ * @property {function} onClick - callback die afgevuurd wordt na click op map
+ */
+
+// TODO: crs doorsturen? vb: crs=EPSG:31370 => komt uit map??
+export interface ClickCallback {
+    coordinate: Coordinate,
+    pixel: Pixel,
+    currentBoundingBox: Extent
+}
+
+
+//TODO: interactie met controls? (zoals current location)
+//TODO: wat als geklikt wordt op zelfde locatie van een feature?
+@webComponent('vl-map-click-action')
+export class VlMapClickAction extends LitElement {
+
+
+    /**
+     * TODO: open tot discussie
```

Comment:
Te bediscussieren

Reply:
Ik ga het omzetten naar events. Dat is vrij simpel te bekomen

Reply:
ik ben voorstander om `@event` cfr Lit style te gaan gebruiken. Maar zoals gezegd, vraag voor UIG team.

uiteindelijk is er best ook consistente manier voor de ganse UIG code base